db/schema.rb

```diff
@@ -153,6 +153,7 @@
     t.datetime "created_at"
     t.datetime "updated_at"
     t.boolean  "superapp",     default: false, null: false
+    t.string   "website"
```

Comment:
Can we use "url" for consistency across different models? Alternatively: "website_url"